### PR TITLE
BK-7666: Upgrade Scality to version 8.1.0

### PR DESCRIPTION
**Pull request by michaelk-stratoscale**

### CODE REVIEW

**Review by chenyosef — APPROVED** [review]
LGTM

File: Dockerfile.s3-scality [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
-    && apt-get install -y jq python git build-essential --no-install-recommends \
+    && apt-get install -y jq python git curl build-essential --no-install-recommends \
     && npm install --production \
-    && apt-get autoremove --purge -y python git build-essential \
-    && rm -rf /var/lib/apt/lists/* \
-    && npm cache clear \
     && rm -rf ~/.node-gyp \
-    && rm -rf /tmp/npm-* \
-    && apt-get clean
+    && rm -rf /tmp/npm-*
```
Comment (romfreiman) [inline Dockerfile.s3-scality]: you removed that 'autoremove' and apt-get clean. It will increase the image size

File: config.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
         "bootstrap": ["localhost"]
     },
     "vaultd": {
-        "host": "s3-vault.service.strato",
+        "host": "localhost",
```
Comment (romfreiman) [inline config.json:52]: mmm u have to put here a service-gw port, no?

File: docker-entrypoint.sh [hunk 3]
```diff
@@ -3,13 +3,13 @@
 # set -e stops the execution of a script if a command or pipeline has an error
 set -e
 
-S3_MANAGER_URL="http://s3-manager-api.service.strato:7540/api/v2/object-stores"
+S3_VAULT_URL="http://127.0.0.1:8600/_/healthcheck"
 
-if [[ "$VERIFY_SERVICE_ENABLED" ]]; then
+if [[ "$VERIFY_S3_VAULT_ACCESSIBLE" ]]; then
```
Comment (romfreiman) [inline docker-entrypoint.sh]: All this if is not needed anymore. It was my hack not to start scality before it's enabled by the user.

Comment (romfreiman) [reply docker-entrypoint.sh]: see this commit: 1a148ec775667f115ab4909987c9e450a5f7eab8

Comment (michaelk-stratoscale) [reply docker-entrypoint.sh]: Why would you need scality managed service if there is no vault to authorize requests?

File: config.json (hunk 2 above)
Comment (michaelk-stratoscale) [reply config.json:52]: Next line says:
        "port": 8600

**Pull request closed without merging**